### ISSUE TITLE
Model's save() not work

### ISSUE THREAD

**Issue by KaungZawHtet** [yiisoft/yii2#16173]
It doesn't work due to wrong validation error. Is there any debugging function for why save() not work?

| ---------------- | ---
| Yii version | 2.0.12?
| PHP version | 7.14
| Operating system | Mac

**Comment by FabrizioCaldarelli** [comment]
Have you tried with:

var_dump($model->errors);

after $model->save() ?

**Comment by KaungZawHtet** [comment]
I found this: 'file can't be blank'
Actually I am still dealing with the err mentioned in this link https://github.com/yiisoft/yii2/issues/16163
But still don't know why and still not find the solution

**Comment by FabrizioCaldarelli** [comment]
Have you a rule with 'required' validator applied to a field named 'file' ?

**Comment by KaungZawHtet** [comment]
Yes, only adding 'required' give problem. But there is one thing that according to need, at first page, I do uploading process and at sec page, I do saving process. Does that make me prevent from saving?
And when I debug with var_dump($model->file) , it's empty really. Is file variable removed from memory when it reaches to other page?
For the process of upload-save via 2 page, here is the reference: https://github.com/yiisoft/yii2/issues/16080
I did as that suggestion.

**Comment by FabrizioCaldarelli** [comment]
If you have a 'required' validator applied to 'file' field and you save the model after processing file upload, it is normal that you get the error, because the model is not receiving the file.

You could set the scenario used by the model to require the file only on the first action (where the file is really uploaded).
